pkg/manager/impl/execution_manager.go

```diff
@@ -619,6 +619,12 @@ func (m *ExecutionManager) launchExecutionAndPrepareModel(
 		return nil, nil, err
 	}
 
+	projectLabels := make(map[string]string)
+	project, err := m.db.ProjectRepo().Get(ctx, request.Project)
+	if err != nil {
+		projectLabels = project.labels.values
```

Comment:
you should call the transformer: https://github.com/lyft/flyteadmin/blob/eb9236eea8af44791d87c0f4113b92d0da32fa0f/pkg/repositories/transformers/project.go#L28 in order to get the deserialized label values